### ISSUE TITLE
Accessibility - radios/checks have orphaned labels and missing fieldset

### ISSUE THREAD

**Issue by vollmerr** [vollmerr/state-template#36]
FieldRadioButtons and FieldCheckboxes currently give the following accessibility alerts according to 
 WAVE:

- Orphaned form label (on field groups label)
- Missing fieldset (on fields options)
